integration-cli/docker_cli_experimental_test.go

```diff
@@ -0,0 +1,24 @@
+// +build experimental
+
+package main
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/go-check/check"
+)
+
+func (s *DockerSuite) TestExperimentalVersion(c *check.C) {
```

Comment:
I think you may need an additional test to make sure that "-experimental" **isn't** printed when the build flag isn't set.